src/model_theory/basic.lean

```diff
@@ -35,6 +35,10 @@ structures.
   to the `L`-structure `N` that commutes with the interpretations of functions, and which preserves
   the interpretations of relations in both directions.
 
+## TODO
+
+Use `[countable L.symbols]` instead of `[L.countable]`.
```

Comment:
Should this change be implemented in this PR?
If not, I can make this change soon.

Reply:
I think that this should go to another PR (and be done by someone who understands math in this folder, e.g., by you).